internal/rcm/api.go

```diff
@@ -82,6 +82,42 @@ func notFoundHandler(writer http.ResponseWriter, request *http.Request) {
 	writer.WriteHeader(http.StatusNotFound)
 }
 
+// Depsolves packages and build packages for building an image for a given
+// distro, in the given architecture
+func depsolve(rpmmd rpmmd.RPMMD, distro distro.Distro, repos []rpmmd.RepoConfig, imageType common.ImageType, arch common.Architecture) ([]rpmmd.PackageSpec, []rpmmd.PackageSpec, error) {
```

Comment:
This looks like something that Weldr and RCM APIs should have in common. Why is it not common?

Reply:
`weldr` also takes a blueprint as an input, which the RCM API doesn't know anything about. We could move the low-level function to `rpmmd` indeed, but I couldn't come up with a good name yet.

Reply:
RCM API has a predefined blueprint which is enough. I'd prefer to move this function to `rpmmd`.